### PR TITLE
OCPBUGS#24317 Correcting maxunavailablestatefulset status in RNs

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.14

Issue: [OCPBUGS-24317](https://issues.redhat.com/browse/OCPBUGS-24317)

Link to docs preview: [Preview](https://73706--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-MaxUnavailableStatefulSet)

QE review:
- [X] QE has approved this change.

Additional information: This PR will need Change Management.

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1114,7 +1114,9 @@ For more information, see xref:../nodes/scheduling/nodes-scheduler-pod-topology-
 [id="ocp-4-14-MaxUnavailableStatefulSet"]
 ==== MaxUnavailableStatefulSet enabled
```
Comment (bergerhoffer) [inline release_notes/ocp-4-14-release-notes.adoc:1115]:
```suggestion
==== MaxUnavailableStatefulSet enabled (Technology Preview)
```

**Review by bergerhoffer — COMMENTED** [review]
One suggestion!

**Comment by gauravsingh85** [comment]
ACK

**Comment by ahardin-rh** [comment]
LGTM. Thank you!

**Comment by reestr** [comment]
LGTM

**Comment by ingvagabund** [comment]
LGTM

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Pull request merged**